src/cryptenroll/cryptenroll-wipe.c

```diff
@@ -355,11 +355,12 @@ int wipe_slots(struct crypt_device *cd,
         case WIPE_EXPLICIT:
                 break; /* Nothing to do here */
 
-        case WIPE_ALL:
+        case WIPE_ALL:                
                 r = find_all_slots(cd, wipe_slots, keep_slots);
-                if (r < 0)
+                if (r < 0){
+                        log_warning("You have deleted ALL slots, make sure to register new keys before rebooting");
```

Comment:
`find_all_slots()` fails only on OOM, and the failure is already logged.
The log message sounds irrelevant to the error cause.